resources/[gameplay]/mapstop100/votestop100.lua

```diff
@@ -0,0 +1,154 @@
+handlerConnect = nil
+canScriptWork = true
+addEventHandler('onResourceStart', getResourceRootElement(),
+function()
+	handlerConnect = dbConnect( 'mysql', 'host=' .. get"*gcshop.host" .. ';dbname=' .. get"*gcshop.dbname", get("*gcshop.user"), get("*gcshop.pass"))
```

Comment:
This seems to be double from mapstop100.lua:5 ? This part overwrites that global variable anyway + its recommend to make only one connection per resource